### ISSUE TITLE
typescript definition is missing in npm package

### ISSUE THREAD

**Issue by spoike** [salvoravida/redux-immer#4]
`redux-immer@1.0.3` package is missing the typescript definition file

You can quickly see the actual file contents if you run `npm pack redux-immer` and the `lib/index.d.ts` is missing from the file list as well as in the tarball file. Here is the terminal output:

```
❯ npm pack redux-immer
npm notice
npm notice 📦  redux-immer@1.0.3
npm notice === Tarball Contents ===
npm notice 52B   .babelrc
npm notice 7B    .eslintignore
npm notice 1.1kB LICENSE
npm notice 673B  lib/redux-immer.js
npm notice 438B  src/redux-immer.js
npm notice 484B  .eslintrc.json
npm notice 1.3kB package.json
npm notice 2.6kB README.md
npm notice === Tarball Details ===
npm notice name:          redux-immer
npm notice version:       1.0.3
npm notice filename:      redux-immer-1.0.3.tgz
npm notice package size:  3.0 kB
npm notice unpacked size: 6.6 kB
npm notice shasum:        cf295f598e6e3db86b405dc0590f7ca5efd5a966
npm notice integrity:     sha512-JOQ2uK9vKAg5a[...]7yx0TQX1FEe0Q==
npm notice total files:   8
npm notice
redux-immer-1.0.3.tgz
```

**Comment by spoike** [comment]
Current work-around is to add the definition (from [here](https://github.com/salvoravida/redux-immer/blob/master/lib/index.d.ts)) to your project's `global.d.ts` file, like this:

```
declare module 'redux-immer' {
  /// <reference types="immer" />
  /// <reference types="redux" />
  export function combineReducers<S, A extends Action = AnyAction>(produce: IProduce, reducers: ReducersMapObject<S, A> = {} as ReducersMapObject): Reducer<S, A>;
}
```

**Comment by salvoravida** [comment]
fixed  1.0.4 sorry
